docs/reference/publication_formats.md

```diff
@@ -1,12 +1,152 @@
 # Publication formats reference
 
+```{admonition} Alpha consultation
+The following issues relate to this page:
+* [#51 Packaging multiple networks](https://github.com/Open-Telecoms-Data/open-fibre-data-standard/issues/51)
+```
+
+OFDS data must be published using at least one of the formats described on this page:
+
+* [JSON](#json)
+* [GeoJSON](#geojson)
+* [CSV](#csv)
+
+Each format provides containers for publishing one or more networks and options to support pagination and streaming.
+
+To support the widest range of use cases, you should publish your data in all three formats. For more information on choosing a publication format and on publishing data in multiple formats, see [how to format data for publication](../guidance/publication.md#how-to-format-data-for-publication).
+
 ## JSON
 
+This section describes how to publish data in JSON format.
+
+If your data is small enough to fit into memory or if you are publishing data via an API, you should use the [small files and API responses option](#small-files-and-api-responses-option). If your data is too large to fit into memory, you should use the [streaming option](#streaming-option).
+
+For guidance on paginating or streaming individual networks, see [how to publish large networks](../guidance/publication.md#how-to-publish-large-networks).
+
+### Small files and API responses option
+
+The network package schema describes the structure of the container for publishing one or more networks in JSON format and for supporting pagination.
+
+For this version of OFDS, the canonical URL of the schema is [https://raw.githubusercontent.com/Open-Telecoms-Data/open-fibre-data-standard/0.1.0-alpha/schema/network-package-schema.json](https://raw.githubusercontent.com/Open-Telecoms-Data/open-fibre-data-standard/0.1.0-alpha/schema/network-package-schema.json). Use the canonical URL to make sure that your software, documentation or other resources refer to the specific version of the schema with which they were tested.
```

Comment:
should we add a note saying that this URL isn't yet active?

Reply:
I think it's OK for it to be broken in the pre-alpha. It should work once we tag the alpha release.